reboot/README.md

```diff
@@ -0,0 +1,54 @@
+## Description
+
+The reboot Update Module: reboot your device remotely.
+
+Example use-cases:
+
+* something went wrong and you have to reboot your device remotely, and all
+access to the device you have is Mender.
+
+### Specification
+
+|||
+| --- | --- |
+|Module name| reboot |
+|Supports rollback|no|
+|Requires restart|no|
+|Artifact generation script|yes|
+|Full system updater|no|
+|Source code|[Update Module](https://github.com/mendersoftware/mender-update-modules/tree/master/reboot/module/reboot), [Artifact Generator](https://github.com/mendersoftware/mender-update-modules/blob/master/reboot/module-artifact-gen/reboot-artifact-gen)|
+
+### Install the Update Module
+
+Download the latest version of this Update Module by running:
+
+```
+mkdir -p /usr/share/mender/modules/v3 && wget -P /usr/share/mender/modules/v3 https://raw.githubusercontent.com/mendersoftware/mender-update-modules/master/reboot/module/reboot && chmod +x /usr/share/mender/modules/v3/reboot
+```
+
+This Update Module depends on the `kubectl` CLI.
```

Comment:
Is this a copy/paste thing or does it really require kubectl?

Reply:
ah left that one. copy-paste. thanks.